meteor/lib/rundown/rundownTiming.ts

```diff
@@ -474,6 +501,35 @@ export class RundownTimingCalculator {
 				}
 			}
 
+			let nextSegmentIndex = -1
+			segments.forEach((segment, itIndex) => {
```

Comment:
For the sake of readability I'd suggest rewriting this block as a `for (const [itIndex, segment] of segments.entries())`, so instead of nested ifs you can continue to the next iter, e.g. `if (segmentIsUntimed) continue;`